onnxruntime/core/graph/graph.cc

```diff
@@ -605,17 +621,25 @@ bool Node::TryGetFunctionProto(ONNX_NAMESPACE::FunctionProto& onnx_function_prot
       // as the default-version, which is incorrect in the case of functions belonging to
       // non-onnx domains, like MSDOMAIN.
 
-      // We use the following as a temporary hack.
-      function_ptr = op_->GetFunction(SinceVersion(), false);
-
-      // TODO: Switch to following, once ONNX issue is fixed.
-      // auto& map = graph_->DomainToVersionMap();
-      // const auto iter = map.find(kOnnxDomain);
-      // if (iter != map.end()) {
-      //   function_ptr = op_->GetFunction(iter->second, true);
-      // } else {
-      //   function_ptr = op_->GetFunction();
-      // }
+      auto requested_opset_version = get_opset_version(graph_);
+
+      /// When we fix custom function registration then we can apply the following
+      /// logic for non ONNX domains as we do for ONNX. Currently, we only apply ONNX
+      /// domain version
+      /* {
```

Comment:
I think we can delete this commented out code now. I think the preceding comment (about fixing the custom function registration) now applies to the check in line 586 (that is, that check will not be necessary in the future.)